standalone/Microsoft.Private.Intellisense/xmldocs/netcoreapp/1033/System.Runtime.Numerics.xml

```diff
@@ -148,21 +151,7 @@
       <summary>Compares this instance to an unsigned 64-bit integer and returns an integer that indicates whether the value of this instance is less than, equal to, or greater than the value of the unsigned 64-bit integer.</summary>
       <param name="other">The unsigned 64-bit integer to compare.</param>
       <returns>A signed integer that indicates the relative value of this instance and <paramref name="other" />, as shown in the following table.
-  Return value  
-  
-  Description  
-  
-  Less than zero  
-  
-  The current instance is less than <paramref name="other" />.  
-  
-  Zero  
-  
-  The current instance equals <paramref name="other" />.  
-  
-  Greater than zero  
-  
-  The current instance is greater than <paramref name="other" />.</returns>
+Return valueDescriptionLess than zeroThe current instance is less than <paramref name="other" />.ZeroThe current instance equals <paramref name="other" />.Greater than zeroThe current instance is greater than <paramref name="other" />.</returns>
```

Comment:
@mairaw - is this expected as the intellisense output for tables? I would imagine this looking odd in VS. It makes sense (and looks great) on the docs site.

https://github.com/dotnet/dotnet-api-docs/blob/6d40805a3453d59a5707781cbae4553c82dc2321/xml/System.Numerics/BigInteger.xml#L1133-L1152

https://docs.microsoft.com/en-us/dotnet/api/system.numerics.biginteger.compareto?view=netcore-3.0#System_Numerics_BigInteger_CompareTo_System_UInt64

![image](https://user-images.githubusercontent.com/6527137/70277753-d0c71600-1767-11ea-9083-493cf03cc648.png)

Reply:
It probably doesn't look great. Tables have always been problematic for IntelliSense and it's been in our backlog to fix them (https://github.com/dotnet/dotnet-api-docs/issues/2264).
But the new IntelliSense files that I'm currently testing output something better:

![image](https://user-images.githubusercontent.com/12971179/70281901-0cb2a900-1771-11ea-91fb-d93df034e803.png)